helm/druid/Chart.yaml

```diff
@@ -17,6 +17,10 @@ apiVersion: v2
 appVersion: 0.22.0
 description: Apache Druid is a high performance real-time analytics database.
 name: druid
+maintainers:
```

Comment:
Since @maver1ck @AWaterColorPen and I are the [authors](https://github.com/helm/charts/commit/ebe628b5adb8c5410e844a12650fb829dcfdd594#diff-a29b6218f791902feb80876c36a479a9f1b1211b06e4c99b770e7f67fa53f66c) of this Apache Druid Helm Chart, it is better to add the three of us too. So if there are any questions to discuss in the future, it is also easy to find us

Reply:
I have updated the helm chart maintainers list

Reply:
There is no particular maintainers in apache projects because those projects are maintained by communities which anyone can join if they want. We should not add such a list.

Reply:
@jihoonson Sure, please @isandeep41 help to remove the list, thanks

Reply:
thanks, I have removed the list and added a flag to skip the maintainers validation